apps/src/templates/projects/submitProjectDialog/submit-project-dialog.module.scss

```diff
@@ -0,0 +1,61 @@
+@import "color", "font";
+
+.submitProjectDialog {
+  max-height: calc(95vh - 100px);
+  display: flex;
+  flex-direction: column;
+  background-color: $light_gray_950;;
+  color: white;
+  padding: 2em 3em;
+
+  hr {
+    margin: 1rem 0;
+    border-color: $light_gray_200;
+  }
+  // override any foreground text in the dialog that might be
+  // passed in for the title or body of the dialog
```

Comment:
Out of curiosity which styles is this overriding exactly?

Reply:
From `Typography`.

Reply:
Gotcha, could we do this with custom classname?

Reply:
I tried this at first but didn't seem to work. I'll try again!

Reply:
Updated! Initially, I tried using a custom style for a div that included both `Typography` elements, but using the custom style for each element works.

Reply:
I saw this pattern of override [in `generic-dialog.module.scss`](https://github.com/code-dot-org/code-dot-org/blob/c0dfe7530ffa3633eb0ed8ed35dbb9e28e9812b8/apps/src/lab2/views/dialogs/generic-dialog.module.scss#L10-L26) which I think makes sense since this modal is intended to be used widely. 

Note that I opted to not use `GenericDialog` since the mockup had the placement/type of the buttons in a different layout and didn't include adding a link in the bottom section. I didn't want to update this generic component with concern about impacting other lab2 components - related to
https://docs.google.com/document/d/1emUpEV2QabYhuJQgtJPz2FWDwjO89I-RfK5KPeKGRdI/edit?disco=AAABX8rbHBc